packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.svelte

```diff
@@ -41,6 +42,10 @@ let existingIssuesLink = $state(
 
 $effect(() => contentChange(Boolean(issueTitle || issueDescription)));
 
+onMount(async () => {
+  await window.telemetryTrack(`feedback.${category}FormOpened`);
```

Comment:
I would recommend category to be a property of the event and event named feedback.formOpened

@slemeur WDYT ?